docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,329 @@
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/actuator_associations.png
+    :align: center
+    :scale: 50
+    :alt: Motor to Actuator to CVI Associations
+
+    Motor to Actuator to CVI Associations
+
+If you accidentally put the wrong CVI on a motor you can just switch the motor functions in the actuator outputs section. If for example you accidentally set the front right motor as CVI 2 and the front left motor as CVI 0, then you can just set ESC 1 as Motor 3, and ESC 3 as Motor 1. You could also change the CVIs associated with each module, but we find that this is the most straightforward way to swap connections.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/actuator_associations_swap.png
+    :align: center
+    :scale: 50
+    :alt: Motor to Actuator to CVI Associations with Swap
+
+    Motor to Actuator to CVI Associations with Swap
+
+Spin Testing
+============
+
+.. warning::
+    Please remove all propellers from any module you plan on testing. Failure to do so can result in harm to you or others around you. Further, please ensure that your module is secured to a stationary platform or surface before attempting to spin it. 
+
+Now that the modules and flight controller have been configured, the modules can be tested. This is done in the same Actuators Tab in QGroundControl with the Actuator Testing section.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/sliders.png
+    :align: center
+    :scale: 50
+    :alt: Actuator Testing
+
+    Actuator Testing Section
+
+Enable the actuator testing slider. The modules should arm when the testing slider is enabled. See the note below for more information on how arming works and how to confirm if the modules have armed. Now the sliders will control the modules that they correspond to. Slide each motor slider up individually and confirm that it corresponds to the expected module in the geometry picture. Additionally, confirm that it is spinning in the correct direction.
+
+.. note::
+    For testing, it is important to understand if your module is using :ref:`Advanced Arming <manual_advanced_arming>` with IFCI. If your module is using Advanced Arming with IFCI, then it will need to arm before it can spin for any of these tests. By default, Vertiq modules will typically arm after 10 consecutive packets of commands between 0% and 7.5% throttle. PX4 sends packets at 400hz by default, so it will take 2.5ms for the motors to arm once the actuator testing slider is engaged and the motor sliders have remained at the bottom. Because of that, for the general default settings used on Vertiq modules arming should not be a concern when running these tests. It should be apparent when your module arms because it will play a :ref:`short arming song <arming_song>`. If you do experience issues where the module will not spin but you believe your configurations are correct, check the arming configurations on your module.
```

Comment:
Because of that, for the general default settings used on Vertiq modules, arming should not be a concern when running these tests